src/server/handlers/redirect.js

```diff
@@ -6,7 +6,9 @@ import { log } from '../utilities/logger'
 
 const setRedirects = (server, redirects) => {
   server.ext('onPreHandler', async (request, h) => {
-    const redirect = redirects[request.url.pathname.toLowerCase()]
+    const redirect =
+      redirects[decodeURIComponent(request.url.pathname.toLowerCase())]
```

Comment:
Interesting, why this change?

Reply:
Upgrading Hapi is causing for some random encodings - need to investigate why as it's bugging me. IMO it's safer to decodeURIComponent anyway as it's being used as a key to find the redirect path.